swan-lake/integration-tools/openapi-tool.md

```diff
@@ -285,6 +287,70 @@ $ bal openapi -i <openapi-contract> --mode client --client-methods <resource|rem
 
 >**Info:** For more command options, see [OpenAPI to Ballerina CLI options](#openapi-to-ballerina-command-options).
 
+### Automate client generation with Package Build
```

Comment:
The automation is supported for all the ballerina commands right? - `build`, `pack`, `test`

How about `Automate client generation using the Ballerina tool integration`?

Reply:
Yeah, `build, `pack`, `test` and  `run` commands execute at least some phases of the build process.  That is why here it used `package build`. 
@azinneera , @Dilhasha  will you be able to provide some input for the title we need to add for these tool scenarios?

Reply:
Shall we say just "Automate client generation"?

Reply:
I will add it as the title